documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,29 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (All optional):
+
+* *Preserving the status ID of Windows VMs*: By default, Virtual NICs (VNICs) change during the migration process, which results in Windows VNICs that are set with static IP in vSphere to lose their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, toggle the *Whether to preserve the static IPs* switch in the window that opens, and then click *Save*.
```

Comment:
```suggestion
* *Preserving the status ID of Windows VMs*: By default, Virtual Network Interface Cards (VNICs) change during the migration process, which results in Windows VNICs that are set with static IP in vSphere to lose their IP. To avoid this, click the Edit icon next to *Preserve static IPs*, toggle the *Whether to preserve the static IPs* switch in the window that opens, and then click *Save*.
```

* If you introduce an acronym, then you must give the full definition first.